core/includes/theme.inc

```diff
@@ -1497,6 +1497,33 @@ function theme_has_settings($theme) {
   return FALSE;
 }
 
+/**
+ * Detect if a theme has support for the Color module.
+ *
+ * This utility function checks a theme's .info file for whether support for the
+ * Color module has been enabled. If the theme has a base theme, it loops
+ * through the .info files of the parent themes as well.
+ *
+ * @param string $theme
+ *   The theme name to be checked for Color module support.
+ *
+ * @return bool
+ *   TRUE if the theme supports Color module, FALSE otherwise.
+ *
+ * @since 1.15.0
+ */
+function theme_has_color_support($theme) {
+  $themes = list_themes();
+  $theme_info = $themes[$theme];
+  if ($theme_info->info['settings']['color'] && file_exists(drupal_get_path('theme', $theme_info->name) . '/color/color.inc')) {
```

Comment:
You should use `backdrop_get_path()` instead of `drupal_get_path()`.

Reply:
Oops! 😅 ...fixed 👍